plasma/plasma-mvp/morevp-simplified

```diff
@@ -0,0 +1,62 @@
+# More Viable Plasma
+
+## TL;DR
+
+This document provides a largely simplified specification for More Viable Plasma, a version of [Minimal Viable Plasma](https://ethresear.ch/t/minimal-viable-plasma/426) (“Plasma MVP”) that removes the need for confirmation signatures.
+
+## Background
+
+Transactions in Plasma MVP are processed in time-wise order. The “exit priority” of any UTXO is determined by the block in which the transaction was included, then the index of the transaction within that block, and finally the index of the output in the list of transaction outputs (“output-age priority”). Plasma MVP requires the use of confirmation signatures to prevent the operator from including a valid transaction time-wise after an invalid transaction. Without confirmation signatures, the Plasma MVP exit priority mechanism would process the invalid exit before the valid exit. 
+
+Confirmation signatures are generally bad user experience. Users must sign a transaction, wait for it to be included in a block, and sign a second signature attesting that the transaction was included in a valid block. Although clients with “hot” wallets can quickly (and usually automatically) sign these secondary signatures, clients with “cold” wallets will face an annoying user-flow (e.g. clicking some button on a hardware wallet).  
+
+Confirmation signatures must also be published to the Plasma chain to avoid certain attack vectors. This significantly reduces the amount of block space available for transactions. 
+
+This motivates the search for an alternative exit game that removes the need for confirmation signatures and simultaneously ensures all valid UTXOs can be processed before all invalid UTXOs. 
+
+## More Viable Plasma
+
+More Viable Plasma (“MoreVP”) proposes an exit game that removes the need for confirmation signatures and ensures correct transaction processing order.
+
+### Basic Exit Game
+
+Under this new protocol, we allow owners of both the *inputs or outputs* to any transaction to attempt an exit. 
+
+If the owner of an input `in` to a transaction `tx` wants to exit, the owner must prove that:
+1. `tx` is correctly formed and properly signed.
+2. One of the inputs to `tx` has already been spent in some transaction other than `tx`.
```

Comment:
"Already spent" here means:
- `tx` is not included in the Plasma chain and another spend exists.
- `tx` is included and another spend is included before it.

Reply:
Will clarify this.

Reply:
OK got it now. It's just that this:

> owner of an input in to a transaction tx wants to exit, the owner must prove that: (...) One of the inputs to tx has already been spent in some transaction other than tx.

is slightly counterintuitive at first. Most likely that owner of an input doesn't know if they want to exit their input, they'll usually just see their spend withheld and start the in-flight exit, assuming that by default it will make the spend forced on rootchain. If they're out of luck, someone else will prove the latter part (non-canonicality) and make the input exitable - which is what makes perfect sense, according to our prior discussions.

Reply:
Yeah I wanted to avoid bringing in all of the complexity of whether or not the owner knows if it's even possible to prove the input is non-canonical. 

In the ideal world (let's imagine a very long exit period and cheap gas), the following would happen:
- Owner of output to some in-flight transaction submit exit. Transaction is shown to be non-canonical and owner doesn't get to exit.
- Owner of input now knows in-flight transaction is non-canonical and submits exit on the input.

This way we avoid the specific complexity of Plasma MVP because we're forced to finish the game within a short period of time and with as little gas as possible.

Reply:
ah, I see your point. Still, now, after all our clarifications, I find the two-phase, explicit descritption of the game (with piggybacking etc, or as you're calling it - the implementation of the game) more approachable and intuitive. It seems it might not be easy to describe / explain what's going on without these specifics and gory details